lib/server/services/lhcFill/getAllLhcFillsOrFail.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+const { getLhcFillByNumberOrFail } = require('./getLhcFillByNumberOrFail.js');
+
+/**
+ * Returns the list of lhcFill ids extracted from a comma separated (CSV) list of lhcFill numbers
+ *
+ * @param {number[]} [lhcFills] the CSV list of lhcFill numbers
+ * @return {Promise<null|number[]>} the list of lhcFill ids
+ * @private
+ */
+exports.getLhcFillNumbersOrFail = async (lhcFills) => {
+    if (!lhcFills) {
+        return null;
+    }
+
+    // Error out if any value is NaN
+    if (lhcFills.some(Number.isNaN)) {
```

Comment:
I think this kind of data validation should not go into low-level services, at this point you should be able to expect normal numbers

On a side note, be careful with `Number.isNaN`: https://stackoverflow.com/questions/33164725/confusion-between-isnan-and-number-isnan-in-javascript